matdeeplearn/common/data.py

```diff
@@ -58,7 +59,9 @@ def dataset_split(
 
 
 def get_dataset(
-    data_path, target_index: int = 0, transform_type="GetY", large_dataset=False
+    data_path,
+    transform_list: list = [],
```

Comment:
Do we want this to be
```suggestion
    transform_list: list = ["GetY"],
```

This goes back to the comment where we were discussing the GetY behavior. I am under the impression that if GetY isn't included as a transform, somewhere within our code, there will be a failure because `y` is used. If this is the case, do we want to throw an error here if GetY isn't included? I think in the future it would make sense to change how we approach `y` vs other targets, but for now, we are dependent on using `y`, so I think it would fail, but double check, because maybe I'm wrong

Reply:
I think it should remain `transform_list: List[dict] = []`. Since we instantiate GetY with an argument, and possibly more in the future, the assertion makes more sense for now as the user should specify GetY.